dll/win32/oleaut32/lang/oleaut32_Id.rc

```diff
@@ -0,0 +1,31 @@
+/*
+ * Indonesian resources for oleaut32
+ *
+ * Copyright 2020 Mas Ahmad Muhammad
+ *
+ * This library is free software; you can redistribute it and/or
```

Comment:
I don't think we usually drop the entire copyright like this in resource headers?

Reply:
yes, i just found the file when i try this translation. that also applies to all translation in this apps i think